sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Utils.java

```diff
@@ -16,6 +16,7 @@
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
```

Comment:
this is a new dependency... do we really need it?

Reply:
We need this dependency to be able to convert long time to Instant. 
Other option is to keep the `ChangeFeedMetaData#conflictResolutionTimestamp` to long data type instead of `Instant`. Then we don't need this.

Reply:
we already do some transformations as such, long to Instant, as part of the lease read/write. can you take a look and see if that is usable? the reason I asked about the dependency is because of the more we add, the more we need to worry about compatibility issues when moving forward with those versions and Jackson is a tricky one on that list.

Reply:
Makes sense Milis, let me take a look at the existing code.

Reply:
Did more research on this, and the only other way to expose `Instant` type and format it properly is to implement custom serializer and de-serializer for `ChangeFeedMetadata` which doesn't make sense for a public facing Pojo. 

In existing CFP code, we basically have `String` types for leases, and store the timestamp and other time variables as `String` type which can be done on public surface as well, but not sure how useful it will be for customer. Although it does reduces the chances of error in case of converting long to instant using jackson library.

Reply:
On top of that, `JavaTimeModule` is very common from jackson and is widely used in our azure-sdks: https://github.com/Azure/azure-sdk-for-java/search?l=Java&q=JavaTimeModule
https://github.com/Azure/azure-sdk-for-java/pull/6024

Reply:
Another option to be consistent with `_ts` is to expose it as long value.